next/components/layout.tsx

```diff
@@ -17,7 +21,10 @@ export interface LayoutProps {
 }
 
 export function Layout({ menus, children }: LayoutProps) {
+  const mainContentRef = useRef<HTMLDivElement>(null);
```

Comment:
This mainContentRef is still not really used (I mean it's passed to the `main` element, but for no reason currently). Let's remove it!